modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerIndexForceRebuildTest.java

```diff
@@ -87,10 +93,11 @@
     private static final int LAST_NODE_NUM = GRIDS_NUM - 1;
 
     /**
-     * Set containing names of caches for which index rebuild should be blocked.
-     * See {@link BlockingIndexing}.
+     * Map for blocking index rebuilds in a {@link BlockingIndexing}.
+     * To stop blocking, need to delete the entry.
+     * Mapping: cache name -> future start blocking rebuilding indexes.
      */
-    private static Set<String> cacheNamesBlockedIdxRebuild = new GridConcurrentHashSet<>();
+    private static final Map<String, GridFutureAdapter<Void>> blockRebuildIdx = new ConcurrentHashMap<>();
```

Comment:
what is the purpose for CHM here?

Reply:
To be able to determine that an index rebuild has begun and blocked.
https://github.com/apache/ignite/pull/8837/files/7958ac85b2afeab9694cc20de4429bc9b1edaf18#diff-900cb374f5842544b1291ed247a50356e0d840553b1953ff2d792b50469b090aR390